static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -192,6 +212,116 @@ function BreadcrumbItem({
 }
 
+function WebVitalData({
+  replay,
+  frame,
+  expandPaths,
+  onInspectorExpanded,
+  onMouseEnter,
+  onMouseLeave,
+}: {
+  expandPaths: string[] | undefined;
+  frame: WebVitalFrame;
+  onInspectorExpanded: (path: string, expandedState: Record<string, boolean>) => void;
+  onMouseEnter: MouseCallback;
+  onMouseLeave: MouseCallback;
+  replay: ReplayReader | null;
+}) {
+  // TODO: remove test CLS data once SDK is merged and updated
+  const clsFrame = {
+    ...frame,
+    data: {
+      value: frame.data.value,
+      size: frame.data.size,
+      rating: frame.data.rating,
+      nodeIds: [frame.data.nodeIds, 333, 870],
+      attributes: [
+        {value: 0.0123, nodeIds: frame.data.nodeIds ?? [93]},
+        {value: 0.0345, nodeIds: [333, 870]},
+      ],
+    },
+  };
+
+  const {data: frameToExtraction} = useExtractDomNodes({replay});
+  const selectors = frameToExtraction?.get(frame)?.selector;
+
+  const webVitalData = {value: frame.data.value};
+  if (
+    frame.description === 'cumulative-layout-shift' &&
+    // frame.data.attributes &&
+    selectors
+  ) {
+    const layoutShifts: {[x: string]: ReactNode[]}[] = [];
+    for (const attr of clsFrame.data.attributes) {
+      const elements: ReactNode[] = [];
+      attr.nodeIds?.map(nodeId => {
+        return selectors.get(nodeId)
+          ? elements.push(
+              <span
+                key={nodeId}
+                onMouseEnter={() => onMouseEnter(clsFrame, nodeId)}
+                onMouseLeave={() => onMouseLeave(clsFrame, nodeId)}
+              >
+                <ValueObjectKey>{'element'}</ValueObjectKey>
+                <span>{': '}</span>
+                <span>
+                  <SelectorButton>{selectors.get(nodeId)}</SelectorButton>
+                </span>
+              </span>
+            )
+          : null;
+      });
+      // if we can't find the elements associated with the layout shift, we still show the score with element: unknown
+      if (!elements.length) {
+        elements.push(
+          <span>
+            <ValueObjectKey>{'element'}</ValueObjectKey>
+            <span>{': '}</span>
+            <ValueNull>{'unknown'}</ValueNull>
+          </span>
+        );
+      }
+      layoutShifts.push({[`score ${attr.value}`]: elements});
+    }
+    if (layoutShifts.length) {
+      webVitalData['Layout shifts'] = layoutShifts;
+    }
+  } else if (selectors?.size) {
+    const vitalKey = 'element';
+    webVitalData[vitalKey] = (
+      <span>
+        {Array.from(selectors).map(([, key]) => {
+          return (
+            <span
+              key={key}
+              onMouseEnter={() => onMouseEnter(frame)}
+              onMouseLeave={() => onMouseLeave(frame)}
+            >
+              <ValueObjectKey>{'element'}</ValueObjectKey>
+              <span>{': '}</span>
+              <SelectorButton>{key}</SelectorButton>
+            </span>
+          );
+        })}
+      </span>
+    );
+  }
+
+  return webVitalData ? (
```

Comment:
is `webVitalData` ever falsey here? looks like it's an object, so the ternary here shouldn't be needed.